UnoCheck/Checkups/AndroidEmulatorCheckup.cs

```diff
@@ -114,8 +114,15 @@ public override Task<DiagnosticResult> Examine(SharedState history)
 
 								var installedPackages = sdkInstance.Components.AllInstalled(true);
 
-								var sdkPackage = installedPackages.FirstOrDefault(p => p.Path.Equals(me.SdkId, StringComparison.OrdinalIgnoreCase));
-
+								var sdkPackage = installedPackages.FirstOrDefault(p =>
+								{
+									// This will be false if the proccess runs on Rosetta emulation
+									// and will install the wrong emulator (x86_64)
+									// https://github.com/dotnet/runtime/issues/42130
+									return Util.IsArm64
+									? p.Path.Contains("arm", StringComparison.OrdinalIgnoreCase)
```

Comment:
Can you add in the comment here an example of the `p.Path` value? I wonder if we're talking about emulator architecture path or the SDK itself.

Reply:
Thanks, that makes sense. Then let's explicitly look for arm64-v8a (the other arm emulators will likely run slowly anyways). With that change make sure to add in the code comment those sample strings for future refence.

Reply:
@jeromelaban this `arm64-v8a` should be a const? If so do we have a good place for it?

Reply:
I don't know if there is, but you can at least make it a const at the top of this file.